src/components/related-assets/utils.ts

```diff
@@ -36,34 +36,37 @@ const removeHackneyHomesRelatedAsset = (relatedAssets: RelatedAsset[]) => {
   return relatedAssets.filter((relatedAsset) => relatedAsset.id !== hackneyHomesGuid);
 };
 
-export const organiseRelatedAssetsByType = (relatedAssets: RelatedAsset[]): Object => {
+const assetTypeSorter = (a: string, b: string) => {
+  if (a === "Dwelling" && b !== "Dwelling") {
+    return 1; // "Dwelling" should be considered greater, so it comes last
```

Comment:
I don't think comments on lines 41 and 44 are meant to be the same, are they?

Reply:
I can remove

Reply:
No no I like the comments, I think they're helpful in this case, but they're the exact same. From the context it doesn't seem like they're meant to be. Shouldn't the comment on line 41 say "... so it comes first" ?

Reply:
I replaced it with a single comment